docs/feature-gates.md

```diff
@@ -55,6 +56,13 @@ manifest provided as part of releases enables this feature by default. If you
 edit the manifest, make sure you do not disable it, as it is needed for correct
 NetworkPolicy implementation for Pod-to-Service traffic.
 
+### EndpointSlice
+
+`EndpointSlice` enable Service EndpointSlice support in Antrea proxy. The
+EndpointSlice API was introduced in Kubernetes 1.16 (alpha) and it is enabled
+by default in Kubernetes 1.17 (beta). This flag will take no effect if AntreaProxy
+is not enabled.
+
```

Comment:
do we support the `Serving` (https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#serving) and `Terminating` (https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#terminating) endpoint states? If not, probably worth documenting here.

you mention in the commit message that ServiceTopology is not supported. Do you mean https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#topology ? It's also probably something worth mentioning here. Because the feature seems deprecated, it may not be something we want to implement in AntreaProxy at all?

Reply:
Currently, the `Serving` and `Terminating` are not supported as they were released with Kubernetes 1.20 alpha. And we will consider supporting them later. `ServiceTopology` is not supported, and we shouldn't implement it in Antrea as it is marked as deprecated. This situations worth documenting here, thanks for suggestion.